### ISSUE TITLE
VS Matt 3.0 With Dialogue

### ISSUE THREAD

**Issue by Kaique62** [luckydog7/Funkin-android#1260]
Download: https://www.mediafire.com/file/s6o6jrwgk6lwpfv/Vs+Matt+3.0.apk/file
Screenshots: 
![image](https://user-images.githubusercontent.com/81991014/121728254-66f99c00-cac3-11eb-9572-1b00bb511c10.png)
![image](https://user-images.githubusercontent.com/81991014/121728266-6bbe5000-cac3-11eb-8440-0aec89a7434c.png)
![image](https://user-images.githubusercontent.com/81991014/121728277-724cc780-cac3-11eb-87cf-bfd6d2093984.png)

**Comment by ROBIKALIVE13** [comment]
pog

**Comment by Blazeplayz08** [comment]
Pog

**Comment by mcagabe19** [comment]
> Download: https://www.mediafire.com/file/s6o6jrwgk6lwpfv/Vs+Matt+3.0.apk/file
> Screenshots:
> ![image](https://user-images.githubusercontent.com/81991014/121728254-66f99c00-cac3-11eb-9572-1b00bb511c10.png)
> ![image](https://user-images.githubusercontent.com/81991014/121728266-6bbe5000-cac3-11eb-8440-0aec89a7434c.png)
> ![image](https://user-images.githubusercontent.com/81991014/121728277-724cc780-cac3-11eb-87cf-bfd6d2093984.png)

optimized?

**Comment by Kaique62** [comment]
> > Download: https://www.mediafire.com/file/s6o6jrwgk6lwpfv/Vs+Matt+3.0.apk/file
> > Screenshots:
> > ![image](https://user-images.githubusercontent.com/81991014/121728254-66f99c00-cac3-11eb-9572-1b00bb511c10.png)
> > ![image](https://user-images.githubusercontent.com/81991014/121728266-6bbe5000-cac3-11eb-8440-0aec89a7434c.png)
> > ![image](https://user-images.githubusercontent.com/81991014/121728277-724cc780-cac3-11eb-87cf-bfd6d2093984.png)
> 
> optimized?

no idk how to optimize it

**Comment by MelvinPlayz** [comment]
Pog

**Comment by MrTks216** [comment]
Man, i wish i have pc, so i can port some mod with coding, instead replace week, rip me. oof

**Comment by JoseGithubFNF** [comment]
> Man, i wish i have pc, so i can port some mod with coding, instead replace week, rip me. oof

Yea!

**Comment by TankPain** [comment]
> Man, i wish i have pc, so i can port some mod with coding, instead replace week, rip me. oof

U dont need a pc

**Comment by TankPain** [comment]
U can use lucky patcher

**Comment by MrTks216** [comment]
> U can use lucky patcher

Well, do you use lucky patcher to do coding?

**Comment by Kaique62** [comment]
> Man, i wish i have pc, so i can port some mod with coding, instead replace week, rip me. oof

If u have a friend with a pc u can code on your phone and ask him to compile the game

**Comment by MrTks216** [comment]
> > Man, i wish i have pc, so i can port some mod with coding, instead replace week, rip me. oof
> 
> If u have a friend with a pc u can code on your phone and ask him to compile the game

Yeah i know, but i don't have many friends on online and real life, and some friends don't have pc, so rip me 💀

**Comment by toybonnie23** [comment]
Optimized

**Comment by luigi98-beep4** [comment]
I have laptop but SLOW i want make faster
